sequenceSearch/celery.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
Remove this file and confirm if it works- let's use the global celery app for all of clustercad